notify/src/lib.rs

```diff
@@ -262,6 +344,34 @@ impl NotifyController {
         let _ = self.new_transaction_notifier.send(tx_entry);
     }
 
+    /// TODO(doc): @quake
+    pub fn subscribe_proposed_transaction<S: ToString>(
+        &self,
+        name: S,
+    ) -> Receiver<PoolTransactionEntry> {
+        Request::call(&self.proposed_transaction_register, name.to_string())
+            .expect("Subscribe new transaction should be OK")
```

Comment:
panic messages should be "Subscribe proposed transaction ...".
And also `subscribe_reject_transaction`